### ISSUE TITLE
Extra triangles appears in decorations

### ISSUE THREAD

**Issue by mdastous-bentley** [iTwin/itwinjs-core#5653]
Extra triangles appears in the displayed decorations.  Can only reproduced when using a blank connection.

![image](https://github.com/iTwin/itwinjs-core/assets/8309609/03dea098-34a2-48da-bee4-c4c7f0e59584)

- checkout the `geo/MapFeatureInfo_hlite` branch of itwinjs-core repository.
- Build and run display-test-app.
- Open blank connection: 
![image](https://github.com/iTwin/itwinjs-core/assets/8309609/a6e73e5c-2d99-4736-b30d-fa1a1af93ad7)

- Run this first key-in: 
`fdt attach arcgisFeature maplayer https://services7.arcgis.com/nZ2Vb4CUwdo9AIiQ/arcgis/rest/services/Philadelphia2/FeatureServer Philadelphia2F`
- Zoom out and fit the Philadelphia area,
- Zoom-in this narrower area:
![image](https://github.com/iTwin/itwinjs-core/assets/8309609/c17cf513-f3b1-4c88-8677-734a05e7651d)

- Run this key-in: `mlt map feature info`
- Click on polygons to reproduce the issue seen in the first screenshot.

**Comment by markschlosseratbentley** [comment]
@mdastous-bentley is this also blocking your PR? (is the blank connection case common?)

**Comment by mdastous-bentley** [comment]
@markschlosseratbentley Right, I think we can wrap up might my PR without this being fixed immediately.
